src/entity/contentPreference/ContentPreferenceSource.ts

```diff
@@ -20,6 +20,10 @@ export class ContentPreferenceSource extends ContentPreference {
   @Column({ type: 'jsonb', default: {} })
   flags: SourceMemberFlags;
 
+  @Column({ type: 'text' })
+  @Index({ unique: true })
+  referralToken: string;
```

Comment:
Why do we need it on content preference though?
Wouldn't it make more sense to keep it under source_member ship?

Reply:
Since joining a source (squad) will now be the same as follow all of it is now saved to `content_preference` table. Source member will be removed when follow migration is done.

Reply:
What about `role` and flags in that case?

Reply:
they are also added (but in previous PR)

Reply:
@rebelchris as concluded on slack I refactored to flags in 557f1cd8c7441244e24ea2137236867e5dea5f28